airqo-platform/AirQo-frontend
# Locate download space frontend

- What does this PR do?
This PR  is for the end product for the Locate platform.
- What JIRA task is related to this PR?
All the tasks on [this JIRA board](https://airqoteam.atlassian.net/jira/software/projects/LOC/boards/32).
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test out the various features of the locate platform.
- Which end points should are ready for testing?
None
